crates/sui-indexer-alt-e2e-tests/tests/jsonrpc/test_framework/framework.move

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# init --protocol-version 70 --simulator --objects-snapshot-min-checkpoint-lag 2
+
+// This test is checking details about the test runner:
+//
+// (1) It does not support GraphQL queries.
+// (2) Tests will fail if the JSON-RPC query does not contian methods or
+//     params.
+//     - No JSON object.
+//     - Missing params.
+//     - Extra trailing comma (tricky!)
+// (3) Displaying response headers is supported.
+//
+// The test description is at the top because the JSON does not have explicit
+// syntax for comments.
+
+//# run-graphql
```

Comment:
Should this be run-jsonrpc?
Also curious why it was possible to mix up? Should we error if it happens?

Reply:
No this is as intended. We do error if you try to run the wrong kind of query, and in fact this is the test for that (see the comments above this line).